packages/openactive-integration-tests/test/helpers/request-helper.js

```diff
@@ -9,30 +9,51 @@ const c2req = require("../templates/c2-req.js");
 const { bReqTemplates } = require("../templates/b-req.js");
 const ureq = require("../templates/u-req.js");
 
+/**
+ * @typedef {import('./logger').BaseLoggerType} BaseLoggerType
+ * @typedef {import('chakram').RequestMethod} RequestMethod
+ * @typedef {import('chakram').RequestOptions} RequestOptions
+ */
+
 const REQUEST_HEADERS = config.get("sellers.primary.requestHeaders");
 
 const MICROSERVICE_BASE = global.MICROSERVICE_BASE;
 const BOOKING_API_BASE = global.BOOKING_API_BASE;
 const TEST_DATASET_IDENTIFIER = global.TEST_DATASET_IDENTIFIER;
 
 class RequestHelper {
+  /**
+   * @param {BaseLoggerType} logger
+   */
   constructor(logger) {
     this.logger = logger;
   }
 
-  async _request(stage, method, url, params, headers) {
```

Comment:
"params" was actually the JSON body.

"headers" was actually params, which can include headers (e.g. `{ timeout: 1000, headers: {} }`).

So I've renamed these to be more clear and to use chakram.request, which has a fixed function signature, rather than chakram.get/chakram.post/etc